lib/rack/http_streaming_response.rb

```diff
@@ -1,9 +1,12 @@
 require "net_http_hacked"
+require "stringio"
 
 module Rack
 
   # Wraps the hacked net/http in a Rack way.
   class HttpStreamingResponse
+    STATUSES_WITH_NO_ENTITY_BODY = [204, 205, 304].freeze
```

Comment:
Using a `Set` (or a `Hash` of `true` as `puma` does) should be faster.

Reply:
Nice catch